django/applications/catmaid/control/skeletonexport.py

```diff
@@ -1164,19 +1164,18 @@ def __init__(self, parent_id, x, y, z):
     class Skeleton():
         def __init__(self):
             self.nodes = {}  # type: Dict[Any, Node]
-            self.raw_cable = 0
-            self.smooth_cable = 0
-            self.principal_branch_cable = 0
+            self.raw_cable = 0.0
```

Comment:
so it never needs to be converted to float later

Reply:
Using a float for cable length seems reasonable.